src/AsyncGenerator.yml

```diff
@@ -0,0 +1,249 @@
+﻿solution:
+  filePath: ..\..\..\src\NHibernate.sln #wants to resolve from the execuing path.
```

Comment:
I published a new version that has the option for registering files with async extension methods. Also, the `NUnitAsyncCountepartFinder` is now built-in into the core, so no need to write a custom one.
[Here](https://github.com/maca88/AsyncGenerator/blob/master/Source/AsyncGenerator.Tests/ExternalProjects/NHibernate/Configuration.yaml#L188) you can find how to register the `NUnitAsyncCountepartFinder` plugin and [here](https://github.com/maca88/AsyncGenerator/blob/master/Source/AsyncGenerator.Tests/ExternalProjects/NHibernate/Configuration.yaml#L155) is how to register the file with async extension methods.